web/src/i18n/locales/zh-CN.json

```
@@ -2896,6 +2896,20 @@
     "补全价格": "补全价格",
     "缓存读取价格": "缓存读取价格",
     "缓存创建价格": "缓存创建价格",
+    "缓存创建价格-5分钟": "缓存创建价格-5分钟",
+    "缓存创建价格-1小时": "缓存创建价格-1小时",
+    "缓存创建价格（5分钟）": "缓存创建价格（5分钟）",
+    "缓存创建价格（1小时）": "缓存创建价格（1小时）",
+    "分时缓存 (Claude)": "分时缓存 (Claude)",
+    "通用缓存": "通用缓存",
+    "缓存读取": "缓存读取",
```

Biome (2.4.11)

[error] The key 缓存读取 was already declared.

(lint/suspicious/noDuplicateObjectKeys)